src/ToDo/Data/TaskData.cs

```diff
@@ -36,4 +40,7 @@ public class TaskData
 
 	[JsonPropertyName("@odata.etag")]
 	public string? OdataEtag { get; set; }
+
+	[JsonPropertyName("parentList")]
+	public ParentListTaskData? ParentList { get; set; }
```

Comment:
If you just need the parent list id, I think you can use parentList/Id as the property name and have it map directly to an ParentListId property

Reply:
It's okay, I'll test

Reply:
I just tried this and it didn't seem to work. So let's leave as is for the moment